### PR TITLE
PROGRAMMES-5743 - Link location stats implementation

### PR DESCRIPTION
**Pull request by Tasiobg**
This PR enables istats tracking and adds or updates the brand pages to meet the ticket requirements.

Notes:
istats-tracking.js is same we have in V2 with a small update, I have added "hardcodedItems" function to track item where we can't add the "data-linktrack" attribute

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/DsAmen/Organism/CoreEntity/CollapsedBroadcast/SubPresenter/CtaPresenter.php [hunk 1]
```diff
@@ -21,7 +21,7 @@ class CtaPresenter extends BaseCtaPresenter
 
     protected $additionalOptions = [
         'link_to_start' => false,
-        'link_location_prefix' => 'programmeobject_',
+        'link_location_suffix' => '',
```
Comment (BjornTwachtmann) [inline src/DsAmen/Organism/CoreEntity/CollapsedBroadcast/SubPresenter/CtaPresenter.php]: This shouldn't need to be here if it's being set in the base presenter. This should only ever be instantiated from there.

Comment (Tasiobg) [reply src/DsAmen/Organism/CoreEntity/CollapsedBroadcast/SubPresenter/CtaPresenter.php]: This is the suffix which is only used on this class, should it be moved to the base class? not sure if any other class will need it

Comment (BjornTwachtmann) [reply src/DsAmen/Organism/CoreEntity/CollapsedBroadcast/SubPresenter/CtaPresenter.php]: Shit. Misread that. Ignore me.

**Pull request merged**